app/Http/Controllers/Users/SellerController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers\Users;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\Users\Seller;
+
+class SellerController extends Controller
+{
+    private $seller;
+
+    public function __construct(Seller $seller)
```

Comment:
![image](https://github.com/kredo-school/18tha-gj_mall/assets/62535075/9883c830-8c13-4404-bd2e-082203ab3c93)

This part of the code is not being used. You may want to remove it and just add it when you use it.